--- a/recommonmark/transform.py
+++ b/recommonmark/transform.py
@@ -251,7 +251,7 @@
                     0, node=node, match_titles=True)
                 return node.children[:]
         else:
-            match = re.search('[ ]?[\w_-]+::.*', language)
+            match = re.search(r'[ ]?[\w_-]+::.*', language)
             if match:
                 parser = Parser()
                 new_doc = new_document(None, self.document.settings)
